toonz/sources/common/twain/ttwain_capability.c

```diff
@@ -273,6 +273,7 @@ int TTWAIN_SetCap(TW_UINT16 cap_id, TW_UINT16 conType, TW_UINT16 itemType,
   if (!capabilityH) {
     GLOBAL_UNLOCK(containerH);
     GLOBAL_FREE(containerH);
+    containerH = NULL;
```

Comment:
if `capabilityH` was not allocated properly, this block will clean up `containerH` allocated in line 266 above, which may get used and freed again in here: https://github.com/opentoonz/opentoonz/pull/558/files/1d0596c8d8043e5905fe897ba2d46df5d3040a8c#diff-e10fd83cafdc05ffcb241ff7619a730dR296